cmd/skupper/skupper.go

```diff
@@ -615,12 +624,17 @@ func NewCmdCreateService(skupperClient SkupperServiceClient) *cobra.Command {
 			if serviceToCreate.EnableTls {
 				serviceToCreate.TlsCredentials = types.SkupperServiceCertPrefix + serviceToCreate.Address
 			}
+			err := serviceToCreate.SetIngressMode(serviceIngressMode)
+			if err != nil {
+				return err
+			}
 
 			return skupperClient.Create(cmd, args)
 		},
 	}
 	cmd.Flags().StringVar(&serviceToCreate.Protocol, "protocol", "tcp", "The mapping in use for this service address (tcp, http, http2)")
 	cmd.Flags().StringVar(&serviceToCreate.Aggregate, "aggregate", "", "The aggregation strategy to use. One of 'json' or 'multipart'. If specified requests to this service will be sent to all registered implementations and the responses aggregated.")
+	cmd.Flags().StringVar(&serviceIngressMode, "ingress-mode", "", "Controls whether a kubernetes service is created locally for the skupper service. Valid values are Always (default) or Never.")
```

Comment:
--ingress-mode doesn't quite fit something with "always" and "never" for me.  Do you conceive of it having any other modes?

Perhaps --create-kube-service (always|never)?   I know it's a bit verbose.  Maybe --create-ingress (always|never).

Reply:
> Do you conceive of it having any other modes?

Possibly. The original PR had a 'IfNoLocalTargets' option as well, but I removed that for simplicity, and as that is a somewhat speculative mode.

Reply:
Maybe `--enable-access-from-target-site`? 'Determines whether access to the Skupper service is enabled in the site the target is exposed in'. (@pwright is this any clearer in your view?)

Reply:
@grs yes, or `create-service-in-target-site` ? 'Determines whether a Skupper service is created in target site'

Reply:
I like that `--enable-access-from-target-site` is not Kube-specific.  "Determines whether access to the Skupper service is enabled in the site the target is exposed in."  The "exposed in" is ambiguous for me.  Where the target is running?

Reply:
Yes, 'where the target is running' is better, but we will soon have the ability to have targets in a namespace other than that which skupper is running in. In those cases we will still create the service in the skupper site namespace. So... what about: "Determines whether access to the Skupper service is enabled in the site the target was exposed through." ?? Any clearer?

Reply:
Yes, that is clearer for me.

Reply:
I went with enable-ingress rather than enable-access, as there are actually two flags involved (one on skupper expose one on skupper service create) and access seemed more vague for the second. Wdyt?